src/backend/opengl/CommandBufferGL.cpp

```diff
@@ -96,6 +97,8 @@ namespace opengl {
                 case Command::BeginRenderSubpass:
                     {
                         commands.NextCommand<BeginRenderSubpassCmd>();
+
+                        glBindFramebuffer(GL_READ_FRAMEBUFFER, 0);
```

Comment:
Added a comment but forgot to put it in this PR. Don't want to restart the CI, so I'll commit it in the next PR.